sklego/linear_model.py

```diff
@@ -49,6 +49,25 @@ class LowessRegression(BaseEstimator, RegressorMixin):
         The training data.
     y_ : np.ndarray of shape (n_samples,)
         The target (training) values.
+
+
+    Example
```

Comment:
A tiny one, but if I remember correctly, rendering will differ

```suggestion
    Examples
```